src/SourceNodes/sourcenode.ts

```diff
@@ -414,7 +414,7 @@ abstract class SourceNode extends GraphNode {
         //update this source nodes texture
         if (this._element === undefined || this._ready === false) return true;
 
-        if (!this._renderPaused && this._state === STATE.paused) {
```

Comment:
_renderPaused prevents a source node from re-updating it's texture if it's already done so for the currentTime. -- obvs if I've drawn into a Canvas I want it to update it's texture next update!